homeassistant/components/homeassistant/strings.json

```
@@ -56,6 +56,10 @@
     "config_entry_reauth": {
       "title": "[%key:common::config_flow::title::reauth%]",
       "description": "Reauthentication is needed"
+    },
+    "template_function_relative_time_deprecated": {
+      "title": "The relative_time() template function is deprecated",
+      "description": "The relative_time() template function is deprecated in Home Assistant. Please use the time_since() or time_until() template functions instead."
```

Inject the name of the functions via translation_placeholders to decrease the risk of translators by mistake translate the function names:

```suggestion
      "title": "The {relative_time} template function is deprecated",
      "description": "The {relative_time} template function is deprecated in Home Assistant. Please use the {time_since} or {time_until} template functions instead."
```